src/sentry/static/sentry/app/components/events/interfaces/debugMeta/debugFileDetails/panel.tsx

```diff
@@ -99,89 +104,96 @@ function Panel({
     <Wrapper>
       <Title>
         {title}
-        <QuestionTooltip title={description} size="xs" position="top" />
+        <QuestionTooltip
+          title={description}
+          size="xs"
+          position="top"
+          isHoverable
+          popperStyle={css`
+            @media (min-width: ${theme.breakpoints[0]}) {
+              max-width: 350px;
+            }
+          `}
```

Comment:
Hmm are you sure this works? You may need to use `import {ClassNames} from '@emotion/core';`

Reply:
yes it works :)